docs/user/add-watermark.md

```diff
@@ -4,80 +4,54 @@ Adding stamps or watermarks are two common ways to manipulate PDF files.
 A stamp is adding something on top of the document, a watermark is in the
 background of the document.
 
-## Stamp (Overlay)
+## Stamp (Overlay) / Watermark(Undelay)
 
-Using the ``Transformation()`` class, one can translate, rotate, scale, etc. the stamp before merging it to the content page.
+actually the process is the same, you just need to set `over` parameter to `True` for Stamping and `False` for watermarking
 
+if you don't need to transform the stamp, you can use `merge_page()`:
 ```python
-from pathlib import Path
-from typing import Union, Literal, List
-
 from pypdf import PdfWriter, PdfReader
 
+stamp = PdfReader("bg.pdf").pages[0]
+writer = PdfWriter(clone_from="source.pdf")
+for page in writer.pages:
+    page.merge_page(stamp, over=False)  # here set to False for watermarking
```

Comment:
I'm not 100% certain, but it's the other way around, right?

```suggestion
    page.merge_page(stamp, over=False)  # here set to False for stamping
```

Reply:
no : over is false so it is for watermarking